docs/api/distribution.md

```diff
@@ -0,0 +1,17 @@
+---
+myst:
+  html_meta:
+    "description": "API methods of module 'distribution'"
+    "property=og:description": "API methods of module 'distribution'"
+    "property=og:title": "plone.distribution.api.distribution"
+    "keywords": "Plone, content, development, API"
+---
+
+(plone-distribution-api-distribution)=
+
+# `plone.distribution.api.distribution`
+
+```{eval-rst}
+.. automodule:: plone.distribution.api.distribution
+    :members:
+```
```

Comment:
When rendered, they look terrible. https://plone.github.io/plone.distribution/api/distribution.html This does not happen in plone.api, for example: https://6.docs.plone.org/plone.api/api/portal.html#plone.api.portal.get

Let's make a note of it and make sure that when it gets imported into the main docs as a submodule that it is addressed.

Reply:
I will try to fix it

Reply:
Fixed now